fighthealthinsurance/common_view_logic.py

```
@@ -527,11 +527,15 @@ def fetch_denial(
     ):
         denial = Denial.objects.filter(
```

Ruff (0.8.2)

`Denial` may be undefined, or defined from star imports

(F405)